src/install.rst

```diff
@@ -46,7 +46,7 @@ These instructions will install the Nextstrain CLI and tools to run and view you
 
          If you are an experienced user, you can replace ``conda`` with ``pip`` but :doc:`note the extra installation steps for Augur <augur:installation/installation>` and :doc:`install Auspice via npm <auspice:introduction/install>`.
 
-      1. Install Miniconda:
+      1. Install Miniconda if you don't have ``conda`` yet:
```

Comment:
A user from Bloom lab just messaged me with some weird behavior where Conda environments weren't being created properly. Turns out they had installed Miniconda on top of an existing Anaconda installation, which then caused weird behaviors. This step should explicitly have users check if there is an existing installation by running `conda` in the command line.